docs/DeveloperGuide.adoc

```diff
@@ -781,14 +781,25 @@ See this https://github.com/se-edu/addressbook-level4/pull/599[PR] for the step-
 == Product Scope
 
 *Target user profile*:
-
+* Freshie Couples/Students in NUS School of Computing
+* - Time Table Sync to NUS Mods in order to find common breaks
+* - Notify on important dates
+* - Journal diary on the meeting
+* - Campus direction shortcuts for new students and when it rains
+* - Couples having diet issues
+* - Location on where to eat
 * has a need to manage a significant number of contacts
```

Comment:
If it's only for couples then why do they need this point? One person can have more than partners?